python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -306,15 +306,15 @@ def makeMinimalSchema(filterNameList, addFluxSigma=False,
                 field = "%s_flux" % (filterName,),
                 type = numpy.float64,
                 doc = "flux in filter %s" % (filterName,),
-                units = "?",
+                units = "count",
             )
         if addFluxSigma:
             for filterName in filterNameList:
                 schema.addField(
                     field = "%s_fluxSigma" % (filterName,),
                     type = numpy.float64,
                     doc = "flux uncertainty in filter %s" % (filterName,),
-                    units = "?",
+                    units = "count",
```

Comment:
Well we're using *some* unit already, and I'm not proposing we change it right now if it's not compliant (but it's much better to leave it blank than set it incorrectly).